src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -417,7 +417,7 @@ instance if $X$ is a type and $A : X → U$ is such that $A x$ is a Segal type f
 all $x$ then $(x : X) → A x$ is a Segal type.
 
 ```rzk title="RS17, Corollary 5.6(i)"
-#def is-segal-function-type uses (funext)
+#def is-segal-function-type' uses (funext)
```

Comment:
It is a bit weird that `is-segal` in the name of a definition actually refers to `is-local-horn-inclusion`. I have been confused by this at least once :)

```suggestion
#def is-local-horn-inclusion-function-type uses (funext)
```

I'm okay with merging this PR as is (since this naming problem is not introduced here), but this point about naming should probably be fixed in the future.